src/popup.js

```diff
@@ -121,6 +121,12 @@ cme.onUninstalled.addListener(id => {
 	$('#' + escapeCssId(id)).remove();
 });
 
+const uiLang = chrome.i18n.getUILanguage();
+const rtlLangs = ['he'];
+$('html')
+	.attr('lang', uiLang)
+	.toggleClass('rtl', rtlLangs.includes(uiLang));
```

Comment:
Does this work? So the rtl languages aren't hardcoded
```suggestion
$('html').attr({
	lang: chrome.i18n.getUILanguage(),
	dir: chrome.i18n.getMessage('@@bidi_dir')
});
```

Reply:
I was happy to learn about `chrome.i18n.getMessage('@@bidi_dir')` 😀